### PR TITLE
Add interface to get inbox for event waiters so that you can integrat…

### PR DESCRIPTION
**Pull request by superchalupa**
…e waits into other select loops.

### CODE REVIEW

File: utils/eventwaiter.go [hunk 1]
```diff
@@ -104,6 +104,10 @@ func (l *EventListener) Wait(ctx context.Context) (eh.Event, error) {
 	}
 }
 
+func (l *EventListener) Inbox() (chan eh.Event) {
```
Comment (maxekman) [inline utils/eventwaiter.go]: Missing doc.

Comment (maxekman) [reply utils/eventwaiter.go]: It would also be nice to return a read only channel in this case.

**Comment by superchalupa** [comment]
repo updated with code changes requested. 1) added doc, 2) return read only channel

**Comment by maxekman** [comment]
Thanks.

**Pull request merged**